src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -95,6 +96,21 @@ public static Email parseEmail(String email) throws ParseException {
         return new Email(trimmedEmail);
     }
 
+    /**
+     * Parses a {@code String organisation} into an {@code Organisation}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code organisation} is invalid.
+     */
+    public static Organisation parseOrganisation(String organisation) throws ParseException {
+        requireNonNull(organisation);
+        String trimmedOrganisation = organisation.trim();
+        if (!Address.isValidAddress(trimmedOrganisation)) {
```

Comment:
Find this bug when deleting the address field. Should be isValidOrganisation here.